client/package.json

```diff
@@ -11,14 +11,22 @@
   "dependencies": {
     "@netlify/plugin-nextjs": "^5.9.2",
     "bonjour-service": "^1.3.0",
+    "d3-ease": "^3.0.1",
     "next": "15.1.3",
     "react": "^19.0.0",
-    "react-dom": "^19.0.0"
+    "react-circular-progressbar": "^2.1.0",
+    "react-colorful": "^5.6.1",
+    "react-dom": "^19.0.0",
+    "react-modal": "^3.16.3",
+    "react-move": "^6.5.0"
   },
   "devDependencies": {
+    "@types/d3-ease": "^3.0.2",
     "@types/node": "^20",
     "@types/react": "^19",
     "@types/react-dom": "^19",
-    "typescript": "^5"
+    "@types/react-modal": "^3.16.3",
+    "typescript": "^5",
+    "usehooks-ts": "^3.1.0"
   }
 }
```

Comment:
can you add overrides to avoid legacy peer deps flag upon npm install
```json
  "overrides": {
    "usehooks-ts": {
      "react": "^19"
    },
    "react-circular-progressbar": {
      "react": "^19"
    }
  }
```